src/align.rs

```diff
@@ -168,6 +168,11 @@ impl<T: Copy> AlignedVec64<T> {
 
         self.len = new_len;
     }
+
+    pub fn clear(&mut self) {
+        self.inner.clear();
+        self.len = 0;
```

Comment:
```suggestion
        // Note: Dropping `T`s not needed because `T: Copy`.
        self.len = 0;
        self.inner.clear();
```

That said, I don't think we need `.clear()` yet (see below).

Also, I'm not really sure if it matters, but setting `self.len = 0` first seems safer in case `.clear()` panics or something.